pipeline.ipynb

```diff
@@ -216,12 +218,14 @@
    "outputs": [],
    "source": [
     "dpath = os.path.abspath(dpath)\n",
+    "hv.notebook_extension('bokeh')\n",
```

Comment:
can we put this into a separate PR? we can deprecate `interactive` completely if we're sure bokeh is working fine in all cases.

Reply:
I would like to have "interactive" to run the notebook without the intermediate results to speed up the result (by skipping display only computations)

Reply:
That sounds good, but I think this should belong to a separate PR since it's independent of the hanging issue? Other than this I think it's ready to merge!